xml/System.Buffers/SequenceReaderExtensions.xml

```diff
@@ -19,11 +19,11 @@
   </Docs>
   <Members>
     <Member MemberName="TryReadBigEndian">
-      <MemberSignature Language="C#" Value="public static bool TryReadBigEndian (this ref System.Buffers.SequenceReader&lt;byte&gt; reader, out short value);" />
-      <MemberSignature Language="ILAsm" Value=".method public static hidebysig bool TryReadBigEndian(valuetype System.Buffers.SequenceReader`1&lt;unsigned int8&gt;&amp; reader, [out] int16&amp; value) cil managed" />
-      <MemberSignature Language="DocId" Value="M:System.Buffers.SequenceReaderExtensions.TryReadBigEndian(System.Buffers.SequenceReader{System.Byte}@,System.Int16@)" />
-      <MemberSignature Language="VB.NET" Value="&lt;Extension()&gt;&#xA;Public Function TryReadBigEndian (ByRef reader As SequenceReader(Of Byte), ByRef value As Short) As Boolean" />
-      <MemberSignature Language="C++ CLI" Value="public:&#xA;[System::Runtime::CompilerServices::Extension]&#xA; static bool TryReadBigEndian(System::Buffers::SequenceReader&lt;System::Byte&gt; % reader, [Runtime::InteropServices::Out] short % value);" />
+      <MemberSignature Language="C#" Value="public static bool TryReadBigEndian (this ref System.Buffers.SequenceReader&lt;byte&gt; reader, out long value);" />
```

Comment:
I think this is still wrong...

I see three overloads for this API:
![image](https://user-images.githubusercontent.com/12971179/61481854-05b80500-a94e-11e9-8a4d-87f623694b4e.png)

Docs is only showing one:
https://review.docs.microsoft.com/en-us/dotnet/api/system.buffers.sequencereaderextensions.tryreadbigendian?view=netcore-3.0&branch=mdoc-dev-test3#System_Buffers_SequenceReaderExtensions_TryReadBigEndian_System_Buffers_SequenceReader_System_Byte__System_Int64__

Reply:
@mairaw can you tell me specifically which assembly (in which moniker) you're looking at to see this?

Reply:
Here: https://apidrop.visualstudio.com/_git/binaries?path=%2Fdotnet%2Fnetcore-3.0%2FSystem.Memory.dll&version=GBnetcore30

Reply:
@mairaw hmm ... ok, so I just re-ran the test locally with the latest version of the `netcore30` branch, and I'm not sure what change may have fixed it, but it appears to be fixed :D As you can see below, there are three overloads in the ecmaxml for each method now:  
![image](https://user-images.githubusercontent.com/90380/61970075-041fba00-afaa-11e9-91ab-70f0f222589d.png)

so I'm going to resolve this conversation, and we can check it on the next release